apps/console/src/extensions/components/password-history-count/components/password-history-count.tsx

```diff
@@ -72,7 +72,7 @@ export const generatePasswordHistoryCount = (
                 <Icon name="info circle" />
                 { t("extensions:manage.serverConfigurations.passwordHistoryCount.message") }
             </Message>
-            <Divider hidden />
+            <Divider className="mt-6 mb-6" />
```

Comment:
AFAIK, we are getting away from semantic ui classes. Let's use Oxygen UI divider and styles here if possible